### PR TITLE
Name actors as per DSDEEPB-1009. Didn't bother w/ the per request

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by geoffjentry** [comment]
It would appear that I forgot ValidateActor, and I should probably not skip the PerRequest stuff

**Comment by scottfrazer** [comment]
Will this make those Akka URIs more readable?  If so that's very nice.

**Comment by geoffjentry** [comment]
Probably, depending on what you mean exactly. The real impetus is for monitoring tools though

**Comment by geoffjentry** [comment]
Actually ValidateActor only is ever called as a PerRequest.

**Comment by kshakir** [comment]
I haven't verified what this looks like in the logs, but scala syntax all looks great.

:+1: for merge from me. Officially throwing over to @scottfrazer for second review.

File: src/main/scala/cromwell/engine/CallActor.scala [hunk 1]
```diff
@@ -99,6 +99,19 @@ class CallActor(call: Call, locallyQualifiedInputs: CallInputs, backend: Backend
...
+    val executeMessage = CallExecutionActor.Execute(workflowDescriptor.id,
+                                                    backend,
+                                                    commandLine,
+                                                    workflowDescriptor,
+                                                    call,
+                                                    backendInputs,
+                                                    inputName => locallyQualifiedInputs.get(inputName).get)
+    executionActor ! executeMessage
+  }
+
```
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/CallActor.scala]: oh my!  conflicts on my SGE branch :( oh well.

**Pull request merged**